go.sum

```diff
@@ -278,7 +278,12 @@ github.com/google/btree v0.0.0-20180813153112-4030bb1f1f0c/go.mod h1:lNA+9X1NB3Z
 github.com/google/btree v1.0.0/go.mod h1:lNA+9X1NB3Zf8V7Ke586lFgjr2dZNuvo3lPJSGZ5JPQ=
 github.com/google/btree v1.0.1 h1:gK4Kx5IaGY9CD5sPJ36FHiBJ6ZXl0kilRiiCj+jdYp4=
 github.com/google/btree v1.0.1/go.mod h1:xXMiIv4Fb/0kKde4SpL7qlzvu5cMJDRkFDxJfI9uaxA=
+<<<<<<< HEAD
```

Comment:
Bad 😄

Reply:
@ncdc what's the proper process to move forward? The cherry-pick creates this merge conflict when the commit is pulled in, should I just take the incoming changes?

Reply:
You can cherry pick but then you have to resolve the conflict

Reply:
Got it, I wasn't sure if it had to be an exact representation of the upstream commit, which seemed unachievable. Thanks!

Reply:
Resolved.

Reply:
I ran `go mod tidy` to resolve these merge conflicts since they were in `go.sum`.

Reply:
Or do some kind of <carry> thing for k8s patch bumps?

Reply:
Resolved offline. Andy and I discussed how we should "stay in the spirit of the commit" if you need to bump different go.mod deps.